<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMsgDialog</class>
 <widget class="QDialog" name="CMsgDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#CMsgDialog
{ 
	background-color: rgb(159,159,159);
	border-radius:15px;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="m_SeackDialog">
     <property name="styleSheet">
      <string notr="true">#m_SeackDialog
{  
	border-radius:15px;
}</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="styleSheet">
       <string notr="true">#page
{  
	border-radius:15px;
}</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="styleSheet">
          <string notr="true">#frame_2
{
	background-color: rgb(255, 255, 255);
	border-radius:15px;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout" stretch="0,2,1,0,0">
          <property name="spacing">
           <number>7</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="mLabel_text2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>65</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel
{
	color: rgb(63, 63, 63);
	font-size:28px;
	border-top-right-radius:15px;
	border-top-left-radius:15px;
}</string>
            </property>
            <property name="text">
             <string>标题</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="mFrame_logo">
            <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0">
             <item row="0" column="0">
              <spacer name="mSpacer_1">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="mLabel_logo">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>9999</width>
                 <height>9999</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>-1</pointsize>
                </font>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="styleSheet">
                <string notr="true">QLabel
{
border-image: url(:/pics/images/插画-机器02.png);
	color: rgb(137, 137, 137);
	font-size:28px;
}</string>
               </property>
               <property name="locale">
                <locale language="Chinese" country="China"/>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <spacer name="mSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,1,0">
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="mLabel_text1">
              <property name="styleSheet">
               <string notr="true">QLabel
{
	color: rgb(63, 63, 63);
	font-size:28px;
}</string>
              </property>
              <property name="text">
               <string>小标题</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QFrame" name="mFrame_Bar">
            <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,6,1">
             <item>
              <spacer name="mSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Minimum</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>13</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QFrame" name="frame">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>44</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>44</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QFrame
{  
	background-color: rgb(13,115,186);
	border-radius:22px;
}</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <property name="leftMargin">
                 <number>2</number>
                </property>
                <property name="topMargin">
                 <number>2</number>
                </property>
                <property name="rightMargin">
                 <number>2</number>
                </property>
                <property name="bottomMargin">
                 <number>2</number>
                </property>
                <item>
                 <widget class="QProgressBar" name="msg_Bar">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QProgressBar{
border:5px solid #ffffff;
height:38;
background:#ffffff;
text-align:center;
color: rgb(63, 63, 63);
border-radius:19px;

font-size:24px;

}
       
QProgressBar::chunk{
border-radius:15px;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:1,y2:0,stop:0 #00929f,stop:1 #22d3e0);
} // 宽度和间距</string>
                  </property>
                  <property name="value">
                   <number>25</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <spacer name="mSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Minimum</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>13</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="mFrame_Btn">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>80</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">#mFrame_Btn
{
	background-color: rgb(233, 236, 245);
	border-bottom-right-radius:15px;
	border-bottom-left-radius:15px;
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>4</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>2</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="mBtn_OK">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>65</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>-1</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton
{ 
border-radius:0px;
	color:rgb(63,63,63);
	font-size:28px; 
background-color:rgb(255, 255, 255);
border-bottom-left-radius:15px;


}
QPushButton:hover
{
border-radius:0px;
background-color:rgb(255, 255, 255);
color:rgb(31, 78, 121);
border-bottom-left-radius:15px;
}</string>
               </property>
               <property name="text">
                <string>确定</string>
               </property>
               <property name="autoDefault">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="mBtn_Cancel">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>65</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>-1</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton
{ 
border-radius:0px;
	color:rgb(63,63,63);
	font-size:28px; 
background-color:rgb(255, 255, 255);

border-bottom-right-radius:15px;
}
QPushButton:hover
{
border-radius:0px;
background-color:rgb(255, 255, 255);
color:rgb(31, 78, 121);
border-bottom-right-radius:15px;
}</string>
               </property>
               <property name="text">
                <string>取消</string>
               </property>
               <property name="autoDefault">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="styleSheet">
       <string notr="true">QWidget
{	  
	background-color: rgb(255, 255, 255);
	border-radius:15px;
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,1,0,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="mLab_Edit_text">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>65</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">#mLab_Edit_text
{
	color: rgb(63, 63, 63);
	font-size:28px;
	border-radius:15px;
}</string>
         </property>
         <property name="text">
          <string> 标题</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,0,3,1">
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="mLab_Edit_Name">
           <property name="styleSheet">
            <string notr="true">QLabel
{
	color: rgb(63, 63, 63);
	font-size:28px;
}</string>
           </property>
           <property name="text">
            <string>批号</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="mEdit_Edit">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>56</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>56</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit
{
	border: 1px solid #e1e3e7;         
	border-radius: 5px;
	font-size:28px;
	color: rgb(137, 137, 137);
	background-color: rgb(255, 255, 255);


}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QFrame" name="mFrame_Edit_Btn">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">#mFrame_Edit_Btn
{
	background-color: rgb(233, 236, 245);
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="mBtn_Edit_OK">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>65</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>-1</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{ 
border-radius:0px;
	color:rgb(63,63,63);
	font-size:28px; 
background-color:rgb(255, 255, 255);
border-bottom-left-radius:15px;



}
QPushButton:hover
{
border-radius:0px;
background-color:rgb(255, 255, 255);
color:rgb(31, 78, 121);
border-bottom-left-radius:15px;
}</string>
            </property>
            <property name="text">
             <string>导入</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="mBtn_Edit_Cancel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>65</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>-1</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{ 
border-radius:0px;
	color:rgb(63,63,63);
	font-size:28px; 
background-color:rgb(255, 255, 255);
border-bottom-right-radius:15px;



}
QPushButton:hover
{
border-radius:0px;
background-color:rgb(255, 255, 255);
color:rgb(31, 78, 121);
border-bottom-right-radius:15px;
}</string>
            </property>
            <property name="text">
             <string>取消</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <property name="styleSheet">
       <string notr="true">QWidget
{	
	background-color: rgb(255, 255, 255);
	border-radius:15px;
}</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
